chains/v5/chains_dev.json

```diff
@@ -2389,8 +2389,8 @@
         ],
         "nodes": [
             {
-                "url": "wss://acala-mandala.api.onfinality.io/public-ws",
-                "name": "OnFinality node"
+                "url": "wss://mandala-rpc.aca-staging.network/ws",
```

Comment:
Their chain Id have updated too
<img width="1624" alt="Screenshot 2022-10-03 at 17 42 49" src="https://user-images.githubusercontent.com/40560660/193579397-a2c23c43-e7e9-4a7d-a985-c31a45e65a9e.png">
Types file should be regenerated too, I suppose